### PR TITLE
Makes v3 the default, merge aligned with LLD/LLM prs

### PR DESCRIPTION
**Pull request by juan-cortes**
Both https://github.com/LedgerHQ/ledger-live-desktop/pull/3854 and https://github.com/LedgerHQ/ledger-live-mobile/pull/1745 need to depend on a live-common version including this switch before they can be merged.

### CODE REVIEW

**Comment by gre** [comment]
am i right to understand we want to merge this only when it's a GO? then i think we can tmp close the PR @juan-cortes

**Comment by juan-cortes** [comment]
> am i right to understand we want to merge this only when it's a GO? then i think we can tmp close the PR @juan-cortes

Sure, close if it's clearer for you 🤷🏼

**Review by valpinkman — APPROVED** [review]

**Comment by gre** [comment]
so when we have a go to wyre feature, we merge it on desktop/mobile with this? @juan-cortes

**Comment by juan-cortes** [comment]
> so when we have a go to wyre feature, we merge it on desktop/mobile with this? @juan-cortes

That is correct, we need this in a live-common version so that it's v3 prod by default. I also had to add a fix for a small bug that triggered the open app twice when we required the latest firmware, it didn't break the flow but it required an extra ok from the user, which is not great. Sorry about that

**Pull request merged**
